pkg/services/state.go

```diff
@@ -145,19 +145,20 @@ func (once *StateMachine) StopOnce(name string, fn func() error) error {
 }
 
 // State retrieves the current state
-func (once *StateMachine) State() State {
-	state := once.state.Load()
-	return State(state)
+func (once *StateMachine) State() string {
```

Comment:
After some tweaks on the core side, the raw state doesn't need to be exposed anymore, but the string format is still used.